src/springtime/datasets/e_obs.py

```diff
@@ -178,17 +179,32 @@ class EOBSBoundingBox(EOBS):
     Fetches complete grid from
     https://surfobs.climate.copernicus.eu/dataaccess/access_eobs.php .
 
+    Example:
+
+        To load coarse mean temperature around amsterdam from 2002 till 2002::
```

Comment:
```suggestion
        To load coarse mean temperature around amsterdam from 2002 till 2002:
```

Reply:
part of google docstring style to render below as code block